content/en/llm_observability/experiments/_index.md

```diff
@@ -152,6 +152,67 @@ dataset.delete(1)  # Deletes the second record
 dataset.push()
 ```
 
+### Dataset versioning
+
+Datasets are automatically versioned to track changes over time. This enables reproducibility and allows experiments to reference specific dataset versions.
+
+#### When are new versions created?
+
+Dataset versions start at 0 and increment automatically when:
+
+- **Adding records**: By default, adding new records creates a new version.
+- **Updating records**: Changes to `input` or `expected_output` fields create a new version
+- **Deleting records**: Deleting records creates a new version
+
+Dataset versions are **NOT** created when:
+
+- **Metadata-only updates**: Updating only the `metadata` field on records or the dataset itself
+- **Dataset changes**: Updating the dataset name or description
+
+#### Version numbering
+
+- Datasets start at version 0 when created
+- Each versioning operation increments the version by 1
+- The `current_version` field in the Dataset object shows the latest version
+
+#### Old versions (retention / TTL)
+
+- **What’s kept:** Only *previous* versions are subject to TTL. The `current_version` is not affected.
+- **How long:** Each previous version has a **90-day retention window**.
+- **Auto-refresh:** **Any use of a previous version resets its 90-day clock** (e.g., experiments that read that version).
+- **Expiration:** If a previous version isn’t used for 90 consecutive days, it becomes eligible for permanent deletion and may no longer be accessible.
+
+**Example**
+
+- You publish `v12`. Now `v11` becomes a previous version with a 90-day window.
+- On day 25, you run an experiment on `v11`. The 90-day window **restarts** from that day.
+- If `v11` isn’t used again for 90 days after that, it may be deleted.
+
+#### Versioning workflow
+
+**Example: Standard versioning workflow**
+
+```python
+# Create dataset (starts at version 0)
+dataset = LLMObs.create_dataset(
+    dataset_name="my-dataset",
+    records=[
+        {"input_data": "test1", "expected_output": "output1"}
+    ]
+)
+print(f"Current version: {dataset.current_version}")  # 0
```

Comment:
```suggestion
print(f"Current version: {dataset._version}")  # 0
```

i think we can add a property to the class to expose the version, can we not include the SDK related docs in this iteration?